lib/definitions/project.d.ts

```diff
@@ -77,12 +72,12 @@ interface IPlatformProjectService {
 	prepareProject(): IFuture<void>;
 	prepareAppResources(appResourcesDirectoryPath: string): IFuture<void>;
 	isPlatformPrepared(projectRoot: string): IFuture<boolean>;
-	canUpdatePlatform(currentVersion: string, newVersion: string): IFuture<boolean>;
+	canUpdatePlatform(newInstalledModuleDir: string): IFuture<boolean>;
 	/**
 	* Provides a platform specific update logic for the specified runtime versions.
 	* @return true in cases when the update procedure should continue.
 	*/
-	updatePlatform(currentVersion: string, newVersion: string, canUpdate: boolean, addPlatform?: Function, removePlatform?: (platforms: string[]) => IFuture<void>): IFuture<boolean>;
+	// updatePlatform(currentVersion: string, newVersion: string, canUpdate: boolean, addPlatform?: Function, removePlatform?: (platforms: string[]) => IFuture<void>): IFuture<boolean>;
```

Comment:
In case the method is commented, can you please delete it. Also the implementation should be removed.